docker/xtask.dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM rust:1.76-slim-bullseye as builder
+
+WORKDIR /usr/src/app
```

Comment:
I'm trying to generate a spec file locally:

```
docker run -v ./out/:/usr/src/app --rm -it --platform linux/x86_64 ghcr.io/devwckd/torus-xtask:4b9fbd3 generate-spec gen-replica --out test.spec.json
```

But it's not writing to ./out.

Can you check this? @devwckd 

Also, let's use a smaller workdir path like `/app`, so we can just do `-v ./out:/app`,